.spin/bin/env

```diff
@@ -6,6 +6,8 @@ export JT_ENV=jvm-ce
 
 unset JAVA_HOME
 
+JT_RUBY=$(which ruby)
+
 function jt {
-  ruby "$TRUFFLERUBY_DIR"/tool/jt.rb "$@"
+  $JT_RUBY "$TRUFFLERUBY_DIR"/tool/jt.rb "$@"
```

Comment:
I have an open PR (created today) that uses this for this line:
```
  "$TRUFFLERUBY_DIR"/bin/jt "$@"
```
And everywhere else as well instead of tool/jt.rb. `bin/jt` is specifically for using $SYSTEM_RUBY if set.
Then we only need `export SYSTEM_RUBY=$(which ruby)` above.
OK if I add that in my PR and close this one?

Reply:
Yes that's fine thanks, modify `jt` in `.spin/bin/env` to use `bin/jt`, and set `export SYSTEM_RUBY=$(which ruby)` there as well.